modules/dynamic-operator/plugin/commons/util.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package commons
+
+import (
+	"github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/go-plugin"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"os"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func FromUnstructured(u map[string]interface{}, obj interface{}) error {
```

Comment:
I do not think these should be exported. There should be a higher level API to work with